js/index.js

```diff
@@ -20,14 +20,71 @@ existingFooter.appendChild(copyright);
 const skills =["Javascript", "HTML", "CSS", "Adobe Photoshop", "gitHub"];
 
 const skillsSection = document.getElementById('skills');
-const skillsList = skillsSection.querySelector('u1');
+const skillsList = skillsSection.querySelector('ul');
 
 //For Loop
 for(let i =0; i<skills.length;i++){
     const skillItem  = document.createElement('li');
     skillItem.innerText = skills[i];
     skillsList.appendChild(skillItem);
 }
-//console.log('Skills Section:', skillsSection);
-//console.log('Skills List:', skillsList);
-// edit edit edit edit
+
+
+//message handler form submit
+const messageForm = document.forms["leave message"];
+
+messageForm.addEventListener("submit", function (event) {
+    event.preventDefault();
+    
+//get values from the form fields
+const usersName = event.target.usersName.value;
+const usersEmail = event.target.usersEmail.value;
+const usersMessage = event.target.usersMessage.value;
+
+//console log values 
+console.log("Name:", usersName);
+console.log("Email:", usersEmail);
+console.log("Message:", usersMessage);
+
+
+//message section
+const messageSection = document.getElementById("messages");
+const messageList = messageSection.querySelector("ul");
+
+//crete a new list item(li) element 
+const newMessage = document.createElement("li");
+newMessage.innerHTML =
+    `<a href="mailto:${usersEmail}">${usersName}</a> 
+    <span> says: ${usersMessage}</span>`
+;
+
+//console log
+//console.log("New message:", newMessage);
+//console.log("Remove button:", removeButton);
+
+//remove button
+const removeButton = document.createElement("button");
+removeButton.innerText = "Remove";
+removeButton.type = "button";
+
+//add event listener tp the removeButton 
+removeButton.addEventListener("click", function() {
+    const entry = removeButton.parentNode;
+    entry.parentNode.removeChild(entry);
+  
+
+
+
+//reset method
+messageForm.reset();
+
```

Comment:
This reset is inside the removeButton event listener and not the submit button event listener, so the form does not get reset when a new message is added.

Reply:
moved the reset inside remove button event listener